packages/contracts/contracts/core/permission/PermissionLib.sol

```diff
@@ -12,13 +12,15 @@ library PermissionLib {
         GrantWithCondition
     }
 
-    struct ItemSingleTarget {
+    /// @notice Simple permission struct to represent a permission without `where` nor `filter`.
```

Comment:
Can we add /// @param NatSpec on the struct items? I guess they can be copied over from `PermissionManager`